Euphorolog.Services/DTOs/AuthDTOs/SignUpRequestDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Euphorolog.Services.DTOs.AuthDTOs
+{
+    public class SignUpRequestDTO
+    {
+        public string userName { get; set; }
+        public string fullName { get; set; }
+        public string eMail { get; set; }
```

Comment:
`E` and `Mail` are not used anymore as different words hyphenated together. So the property name could just be `email`. Or in C# convention, `Email`.